kroxylicious-filters/kroxylicious-encryption/src/main/java/io/kroxylicious/filter/encryption/inband/KeyContext.java

```diff
@@ -51,6 +51,10 @@ public boolean hasAtLeastRemainingEncryptions(int numEncryptions) {
         return remainingEncryptions.getAndAdd(-numEncryptions) >= numEncryptions;
     }
 
+    public int remainingEncryptions() {
```

Comment:
`hasAtLeastRemainingEncryptions` sounded like what I needed but it mutates the remainingEncryption count.